packages/search/tests/unit/services/navi-report-search-provider-test.js

```diff
@@ -0,0 +1,72 @@
+import Service from '@ember/service';
```

Comment:
maybe we can add a test for the base search provider. to make sure it throws the error when you try to search?